sencha-workspace/packages/slate-cbl/src/view/teacher/DashboardController.js

```diff
@@ -119,6 +138,9 @@ Ext.define('Slate.cbl.view.teacher.DashboardController', {
             demonstratedSkills = demonstration.get('Skills'),
             demonstratedSkillsLength = demonstratedSkills.length, demonstratedSkillIndex = 0, demonstratedSkill,
             loadedCompetency, loadedCompetencySkills, loadedSkill, demonstrationsByStudent, loadedDemonstrations, skillDemonstrationsCell, existingDemonstrationSkill,
+            demonstrationFilterFn = function(loadedSkillDemonstration) {
+                return loadedSkillDemonstration.DemonstrationID == demonstration.getId();
```

Comment:
You could change it if you verify `loadedSkillDemonstration.DemonstrationID` is coming down as an integer like it should. I'm not a fan of using strict comparisons everywhere just for the hell of it though, if one of these integers ended up being encoded as a string because of a screw up on the data later the code can still work, there's no danger in the loose comparison causing a problem in this case, a strict comparison would only serve to make the code less fault tolerant